src/components/BarChart/Bars/hooks/useSetBarHeight.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+
+const useSetBarHeight = (parentElement) => {
+	useEffect(() => {
+		const bars = document.querySelectorAll('.charts__bar');
+		const maxHeight = parentElement.current.clientHeight;
+
+		bars.forEach((element) => {
+			const barValue = +element.getAttribute('value');
+			const barHeight = (maxHeight * barValue) / 100;
+
+			element.style.height = `${barHeight}px`;
```

Comment:
Why do you use px?
Using percents will be a mush simple way.
You don't need to know a parent's height for percents.
And it's okay to set this login into Bars component

Reply:
Your calculations also shouldn't depend on the values from DOM usually. You have the data about bar value in the chartData, so there is no need to take it from the attribute, as you do